README.md

```diff
@@ -98,6 +98,8 @@ If you have success/failure using LLM Foundry on other systems, please let us kn
 | A10-24GB                  | 1.13.1           | 11.7         | :construction: In Progress    |
 | A10-24GB                  | 2.0.1            | 11.7, 11.8   | :construction: In Progress    |
 
```

Comment:
Can we add the AMD MI250 as a line here like | 2.0.1 | RoCm 5.4 |  :construction: In Progress